### PR TITLE
Adding GitHub link to the about page

### PR DESCRIPTION
**Pull request by MattyTheHacker**
Adding a link to this GitHub repo on the about page as per #142.

### CODE REVIEW

File: css/about.scss [hunk 1]
```diff
@@ -131,7 +131,13 @@ h1, h2, h3, h4 {
     h4, i {
       color: #ff0000;
     }
-  }
+  }      
+  
+  .github {
+    h4, i {
+      color: #FFFFFF;
```
Comment (rainestormee) [inline css/about.scss:138]: I was going to comment about consistent casing for hex, but it seems like it's inconsistent elsewhere.
We should probably look into having some consistency and decide to use uppercase or lowercase.

Comment (AlexJBanks) [reply css/about.scss:138]: Even better, I'm planning a tidy up of SCSS and colours in future so this can be done in that

**Comment by AlexJBanks** [comment]
Ty @MattyTheHacker, I'll review in morning

File: css/about.scss (hunk 1 above)
Comment (rainestormee) [reply css/about.scss:138]: So for now could be fine, then eventually resolved in #159 or if you have another issue / milestone for that?

**Comment by rainestormee** [comment]
image
This looks very bright to me

**Comment by MattyTheHacker** [comment]
> image
> This looks very bright to me, would it be better to have it in black so it's more readable against a white background?

Damn, that's my bad. Both the logo and the text was supposed to be black in the first place!

**Comment by rainestormee** [comment]
> > image
> > This looks very bright to me, would it be better to have it in black so it's more readable against a white background?
> 
> Damn, that's my bad. Both the logo and the text was supposed to be black in the first place!
> EDIT: Just updated the text but not actually sure how to get the icon black so if anyone else does then please feel free!

image
Well looks like it's now black -- would it be better to do all of these changes in one commit as opposed to a few commits and a change?

**Review by rainestormee — APPROVED** [review]
Looks fine to me, probably have @AlexJBanks double-check though just in case 👀 

The issue's wording is a bit confusing, it may mean link to https://github.com/CSSUoB as opposed to this exact repo however?

**Review by AlexJBanks — APPROVED** [review]
Code is good, Thanks @MattyTheHacker

**Pull request merged**